x-pack/plugins/ml/public/alerting/preview_alert_condition.tsx

```diff
@@ -0,0 +1,277 @@
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FC, useCallback, useEffect, useMemo, useState } from 'react';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+import {
+  EuiButton,
+  EuiButtonEmpty,
+  EuiCallOut,
+  EuiCode,
+  EuiDescriptionList,
+  EuiFieldText,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiFormRow,
+  EuiHorizontalRule,
+  EuiPanel,
+  EuiSpacer,
+  EuiText,
+} from '@elastic/eui';
+import type { AlertingApiService } from '../application/services/ml_api_service/alerting';
+import { MlAnomalyDetectionAlertParams, PreviewResponse } from '../../common/types/alerts';
+import { composeValidators } from '../../common';
+import { requiredValidator, timeIntervalInputValidator } from '../../common/util/validators';
+import { invalidTimeIntervalMessage } from '../application/jobs/new_job/common/job_validator/util';
+
+export interface PreviewAlertConditionProps {
+  alertingApiService: AlertingApiService;
+  alertParams: MlAnomalyDetectionAlertParams;
+}
+
+const AlertInstancePreview: FC<PreviewResponse['results'][number]> = React.memo(
+  ({ jobIds, timestampIso8601, score, topInfluencers, topRecords }) => {
+    const listItems = [
+      {
+        title: i18n.translate('xpack.ml.previewAlert.jobsLabel', {
+          defaultMessage: 'Job IDs:',
+        }),
+        description: jobIds.join(', '),
+      },
+      {
+        title: i18n.translate('xpack.ml.previewAlert.timeLabel', {
+          defaultMessage: 'Time: ',
+        }),
+        description: timestampIso8601,
+      },
+      {
+        title: i18n.translate('xpack.ml.previewAlert.scoreLabel', {
+          defaultMessage: 'Anomaly score:',
+        }),
+        description: score,
+      },
+      ...(topInfluencers && topInfluencers.length > 0
+        ? [
+            {
+              title: i18n.translate('xpack.ml.previewAlert.topInfluencersLabel', {
+                defaultMessage: 'Top influencers:',
+              }),
+              description: (
+                <ul>
+                  {topInfluencers.map((i) => (
+                    <li key={i.unique_key}>
+                      <EuiCode transparentBackground>{i.influencer_field_name}</EuiCode> ={' '}
+                      {i.influencer_field_value} [{i.score}]
+                    </li>
+                  ))}
+                </ul>
+              ),
+            },
+          ]
+        : []),
+      ...(topRecords && topRecords.length > 0
+        ? [
+            {
+              title: i18n.translate('xpack.ml.previewAlert.topRecordsLabel', {
+                defaultMessage: 'Top records:',
+              }),
+              description: (
+                <ul>
+                  {topRecords.map((i) => (
+                    <li key={i.unique_key}>
+                      <EuiCode transparentBackground>
+                        {i.function}({i.field_name})
+                      </EuiCode>{' '}
```

Comment:
nit: extra `{' '}`

Reply:
as it's an inline element we probably still need it